.github/workflows/unit-tests.yml

```
@@ -38,7 +38,9 @@ jobs:
           SESSION_SECRET: "secret"
           MAGIC_LINK_SECRET: "secret"
           ENCRYPTION_KEY: "secret"
-
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)